commit 87cc9488bd08a5f17318ddd70cbfcdb4182f8b16
Author: Jiri Moskovcak <jmoskovc@redhat.com>
Date:   Thu Mar 11 23:50:47 2010 +0100

    minor fix in abrt-debuginfo-install to make it work with yum >= 3.2.26
    
    - yum repolist has different output, so it needs to bu run
      with additional parameter 'all'

diff --git a/src/Daemon/abrt-debuginfo-install b/src/Daemon/abrt-debuginfo-install
index 0d9401b..521d42a 100755
--- a/src/Daemon/abrt-debuginfo-install
+++ b/src/Daemon/abrt-debuginfo-install
@@ -303,7 +303,7 @@ $debug && echo "build_ids:$build_ids"
 # When we look for debuginfo we need only -debuginfo* repos, we can disable the rest
 # and thus make it faster.
 yum_repo_opts="'--disablerepo=*'"
-for enabled_repo in `yum repolist | grep 'enabled:' | cut -f1 -d' ' | grep -v -- '-debuginfo'`; do
+for enabled_repo in `yum repolist all | grep 'enabled:' | cut -f1 -d' ' | grep -v -- '-debuginfo'`; do
     yum_repo_opts="$yum_repo_opts '--enablerepo=${enabled_repo}-debuginfo*'"
 done
 
